### DISCUSSION TITLE
Fails to create Scope_info

### DISCUSSION THREAD

**Discussion by kmostofi** [Mimetis/Dotmim.Sync#867] [Q&A]
Hi,
I had to recreate the SQL DB and since then Dotmim fails to create the relevant tables. It also doesn't generate a new SQLite database. 

It is a WPF C# project, .net framework 4.8. (I had to downgrade the framework, the company refused to upgrade). It doesn't throw any error message at all.

Any ideas?

Kind regards,
Kamal
`  
                string conStr = DbManagerConnection.SqlConnection();
                SqlSyncProvider serverProvider = new SqlSyncProvider(conStr);

                string[] strSqliteCon = DbManagerConnection.SqliteConnection().Split(new string[] { @"\" }, StringSplitOptions.None);

                // Sqlite Client provider acting as the "client"
                var appDir = Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData);

                bool exists = System.IO.Directory.Exists(appDir + @"\" + strSqliteCon[2]);

                if (!exists)
                    System.IO.Directory.CreateDirectory(appDir + @"\" + strSqliteCon[2]);

                // string dcDir = @"DcPro\dcproV3.db3";
                var dbFile = System.IO.Path.Combine(appDir, strSqliteCon[2] + @"\" + strSqliteCon[3]);

                SqliteSyncProvider clientProvider = new SqliteSyncProvider(dbFile);

                // Tables involved in the sync process:
                var setup = new SyncSetup("tblLupFleetM", "tblFleetM");

                // Sync agent
                SyncAgent agent = new SyncAgent(clientProvider, serverProvider);

                var syncContext = agent.SynchronizeAsync(setup);`

**Comment by Mimetis** [comment]
No Idea
Try to use a logger to see what's going on; maybe

``` cs
        var options = new SyncOptions();
        options.Logger = new SyncLogger().AddDebug().SetMinimumLevel(LogLevel.Trace);

```

**Reply by kmostofi** [reply]
Tried it with logger and it didn't give me much information. ScopeInfoTableName is Scope_info. 

Could it be related to any .net framework?

**Reply by Mimetis** [reply]
Maybe.
Try you sample using another framework, like .net 6 and see if it's working as expected

**Reply by kmostofi** [reply]
The project was in .net 6 and it was working. The company asked me to downgrade it to .Net 4.8 4 weeks ago. At the time, I didn't check the code. 
Today, I recreated the database then I noticed the issue which fails to do the following:
- Create the scope tables
- Create the SQLite database

:-(

**Reply by kmostofi** [reply]
What version of Domim is compatible with .Net 4.8?

**Reply by Mimetis** [reply]
No idea :(
